### PR TITLE
First draft of github + trello usage guide

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: code-review/README.md [hunk 1]
```diff
@@ -91,3 +91,4 @@ Ruby on Rails review
 * Review whether new scheduler (`cron`) tasks have been added and whether there
   is a related todo in the project management system to add it during the
   staging and production deploys.
+* Review whether there are new requirements being added to the production or staging machine. This includes new software packages, new secret keys in the app's environment, background services, network configurations, etc.
```
Comment (zamith) [inline code-review/README.md]: This is not indented according to the rest of the file.

Other than that, what should I do, if I find there are new things added?

File: github/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Github
+======
+
+A guide for coding socially and responsibly with Github.
+
+General usage
+-------------
+
+* Github is not only for developers. If you're a Designer or Project Manager or even a tester, you might be requested in a discussion
```
Comment (zamith) [inline github/README.md]: The tester could be capitalized as well.

Comment (naps62) [inline github/README.md]: fixed

File: code-review/README.md (hunk 1 above)
Comment (naps62) [inline code-review/README.md]: fixed the indentation

If you see new things added, you might want to make sure that someone (either the author of the PR or the SysAdmin) already made the required changes to the server before merging this
I updated the text to include this as well

**Comment by naps62** [comment]
@rmdmachado can you review this as well?

**Comment by rmdmachado** [comment]
Looks good.

File: code-review/README.md (hunk 1 above)
Comment (zamith) [inline code-review/README.md]: Oh ok, then what about:

>  Review whether there are new dependencies that will require changes to the production or staging machine(s).

**Pull request merged**